cmd/opera/launcher/defaults.go

```diff
@@ -28,22 +29,27 @@ func overrideFlags() {
 }
 
 // NodeDefaultConfig contains reasonable default settings.
```

Comment:
Godoc agreement: the comment should begins with the name of the element it describes.

Reply:
Updated and also fixed tests